lib/sycamore/sycamore/transforms/detr_partitioner.py

```diff
@@ -124,16 +124,22 @@ def _supplement_text(inferred: list[Element], text: list[Element], threshold: fl
             if matched:
                 matches = []
                 full_text = []
+                font_sizes = []
                 for m in matched:
                     matches.append(m)
                     if m.text_representation:
                         full_text.append(m.text_representation)
-
+                        if m.properties.get("font_size") is not None:
+                            font_sizes.append(m.properties.get("font_size"))
                 if isinstance(i, TableElement):
                     i.tokens = [{"text": elem.text_representation, "bbox": elem.bbox} for elem in matches]
 
                 i.data["text_representation"] = " ".join(full_text)
-
+                i.properties["font_size"] = 0
+                if len(font_sizes) > 0:
+                    font_sizes_filtered = [size for size in font_sizes if size is not None]
```

Comment:
Why do we need to filter if we only add if it is not None? We shouldn't need to do the conditional checks twice.

Reply:
makes sense, shouldn't be needed here